Components/input/input.css

```diff
@@ -0,0 +1,124 @@
+.content {
+    grid-area: content;
+    padding: 3rem 1.5rem;
+}
+
+.input__examples-des {
+    padding-top: 2rem;
+}
+
+
+
+.label {
+    display: inline-block;
+    padding: 1rem 0;
+    box-sizing: border-box;
+    color: var(--color--dark);
+}
+
+.label-required-field::after {
+    content: "*";
+    color: var(--primary-color);
+}
+
+.field {
+    width: 100%;
+    display: block;
+    padding: .8rem;
+    border: 1px solid #d4d5d9;
```

Comment:
use variables to define colors

Reply:
Thanks for reviewing! required changes have been made!